### PR TITLE
IOS ForeignObject fix

### PR DESCRIPTION
**Pull request by Zein-Fakhreddine**
## Summary
Fix ForeignObject not being linked correctly to the IOS project data file. This allows you to use the ForeignObject component on IOS. 

## Test Plan
Import and render a ForeignObject component if it doesn't crash then it works.

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |

## Checklist

- [x] I have tested this on a device and a simulator

### CODE REVIEW

File: ios/RNSVG.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// !$*UTF8*$!
+// !$*UTF9*$!
```
Comment (msand) [inline ios/RNSVG.xcodeproj/project.pbxproj:1]: Why this change?

Comment (Zein-Fakhreddine) [reply ios/RNSVG.xcodeproj/project.pbxproj]: @msand my mistake it seems to have automatically changed that. I got rid of it.
